Visual Studio 2019 Version 16.7.1 でビルドエラーになる

## 問題内容

Visual Studio 2019 Version 16.7.1 でビルドエラーになる

```
3>C:\gitwork\sakura\sakura_core\CGrepAgent.cpp(468,56): error C2039: 'back_inserter': 'std' のメンバーではありません
3>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include\deque(24): message : 'std' の宣言を確認してください
3>C:\gitwork\sakura\sakura_core\CGrepAgent.cpp(468,69): error C3861: 'back_inserter': 識別子が見つかりませんでした
```

## 再現手順
<!-- 該当バグの再現手順を記述ください。 -->
<!-- わかっている範囲で記述ください。 -->

1. Visual Studio 2019 Version 16.7.1  をインストールする
2. sakura.sln を開く
3. ソリューションのビルドを行う

## 再現頻度
2/2

## 問題のカテゴリ

- ビルドの問題
  - ローカルビルド

--- a/sakura_core/CGrepAgent.cpp
+++ b/sakura_core/CGrepAgent.cpp
@@ -20,6 +20,7 @@
 #include "util/module.h"
 #include "util/string_ex2.h"
 #include "debug/CRunningTimer.h"
+#include <iterator>
 #include <deque>
 #include <memory>
 #include "sakura_rc.h"

